### PR TITLE
Use missing and exists filters for null comparisons.

### PR DESCRIPTION
**Pull request by LosD**
This will make isNull() and isNotNull() work. They failed before, as a term query, which does not support null values, was used. A test was also un-ignored; It was ignored as it used to fail, but the fix  for METAMODEL-172 fixed the test as well.

Fixes METAMODEL-184

### CODE REVIEW

File: elasticsearch/src/test/java/org/apache/metamodel/elasticsearch/ElasticSearchDataContextTest.java [hunk 1]
```diff
@@ -559,7 +560,16 @@ private static void indexBulkDocuments(String indexName, String indexType, int n
 
     }
 
-    private static void indexOneTweeterDocumentPerIndex(String indexType, int id) {
+    private static void indexTweeterDocument(String indexType, int id, Date date) {
+        try {
+            client.prepareIndex(indexName, indexType).setSource(buildTweeterJson(id, date))
+                    .setId("tweet_" + indexType + "_" + id).execute().actionGet();
+        } catch (Exception ex) {
+            System.out.println("Exception indexing documents!!!!!");
```
Comment (kaspersorensen) [inline elasticsearch/src/test/java/org/apache/metamodel/elasticsearch/ElasticSearchDataContextTest.java]: Wouldn't a rethrow be better?

**Comment by kaspersorensen** [comment]
LGTM, but just a question regarding the test code.

File: elasticsearch/src/test/java/org/apache/metamodel/elasticsearch/ElasticSearchDataContextTest.java (hunk 1 above)
Comment (LosD) [inline elasticsearch/src/test/java/org/apache/metamodel/elasticsearch/ElasticSearchDataContextTest.java]: I completely agree (except I wouldn't rethrow, but use throws instead), but it seemed to be the pattern for all the indexing methods here, so I just went with the flow :)

**Comment by kaspersorensen** [comment]
+1 from me

**Pull request closed without merging**